tasks.py

```diff
@@ -842,8 +842,11 @@ def e2etest(ctx, name="kind", export=None, kubeconfig=None, system_namespaces="k
     for ns in namespaces:
         run("{} -n {} wait --for=condition=Ready --all pods --timeout 300s".format(kubectl_path, ns), hide=True)
 
+    nodes = run("kind get nodes --name {name}".format(name=name)).stdout.strip().split("\n")
```

Comment:
Can you comment what this new addition does (or even move to it's own function with a descriptive name)?

Reply:
sure, thanks